exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -70,6 +71,9 @@
      */
     GuavaPatcher.patch();
     Environment.logEnv("Drillbit environment: ", logger);
+    // Jersey uses java.util.logging - create bridge: jul to slf4j
+    SLF4JBridgeHandler.removeHandlersForRootLogger();
```

Comment:
Is it for logging info from libs, which `use java.util.logging`? Is the default level suitable for Drill? How the log file is changed?

Does it makes sense to change the log level for it in some cases, possibly via logback.xml?

Reply:
This was done to remove the following, especially when starting Drill in embedded mode.
```
Sep 10, 2018 3:27:01 PM org.glassfish.jersey.server.ApplicationHandler initialize
INFO: Initiating Jersey application, version Jersey: 2.8 2014-04-29 01:25:26...
```
I don't think we need Jersey info logs in Drill log, error will be logged as usual.